kanidmd/score/src/lib.rs

```diff
@@ -390,6 +390,62 @@ pub fn vacuum_server_core(config: &Configuration) {
     };
 }
 
+/// This'll set the domain's display name in the database
+pub fn set_domain_display_name(config: &Configuration, new_display_name: &str) {
```

Comment:
Rather than doing this as an offline operation, why not just add domain_display_name as an access control for writes for admins, and then have a tool in kanidm_tools for it? My thinking is the "displayname" doesn't need offline edit, like you'll never need to change it in a DR to recover the situation (unlike domain name or admin password).